doc/howto/usage/cluster/fluid_cluster_train_en.md

```diff
@@ -0,0 +1,138 @@
+# Fluid Distributed Training
+
+## Introduction
+
+In this article, we'll explain how to config and run distributed training job with PaddlePaddle Fluid in a bare metal cluster.
+
+## Preparations
+
+### Get your cluster ready
+
+Prepare your computer node in the cluster. Nodes in this cluster can be any spec that runs PaddlePaddle, and with a unique IP address assigned to it. Make sure they can talk to each other.
+
+### Have PaddlePaddle installed
+
+PaddlePaddle must be installed on all nodes. It would be great if you have GPU cards on your nodes, be sure to properly install drivers and CUDA libraries.
```

Comment:
Maybe change "It would be great if you have GPU cards" to "If you have GPU cards"? Doesn't seem GPU is any better than CPU only nodes in the distributed settings.

Reply:
let me update, thanks!